src/submission/song.ts

```diff
@@ -194,3 +194,45 @@ export const getAnalysisById = async (organization: string, analysisId: string):
 		throw new Error('Invalid JSON in retriving Analysis response');
 	}
 };
+
+/**
+ * Publishes an Analysis in SONG service
+ * @param organization
+ * @param analysisId
+ * @param ignoreUndefinedMd5
+ * @returns a string if analysis is successfully published. Otherwise, throws an error with details
+ */
+export const publishAnalysis = async (
+	organization: string,
+	analysisId: string,
+	ignoreUndefinedMd5: boolean = false,
+) => {
+	logger.debug(`Publishing analysis ID '${analysisId}'`);
+	const apiUrl = new URL(`/studies/${organization}/analysis/publish/${analysisId}`, env.SEQUENCING_SUBMISSION_URL);
```

Comment:
some blank space to facilitate reading
```suggestion
	const apiUrl = new URL(`/studies/${organization}/analysis/publish/${analysisId}`, env.SEQUENCING_SUBMISSION_URL);

```